Stack/css/styles.css

```diff
@@ -0,0 +1,30 @@
+.values {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.answer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  text-align: center;
+}
+
+.stack, .queue {
```

Comment:
Try to keep your CSS file in one style. Between selectors you have to add 1 empty line. If you need to add 2 selectors for 1 styles, please keep in following style:
```
.stack,
.queue {
...
}
```